src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -361,17 +361,27 @@
 
 (schema/defn create-ca-extensions :- (schema/pred extension-list?)
   "Create a list of extensions to be added to the CA certificate."
-  [ca-name :- (schema/pred valid-x500-name?)
-   ca-serial :- (schema/pred number?)
-   ca-public-key :- (schema/pred public-key?)]
-  [(authority-key-identifier
-     ca-name ca-serial false)
-   (basic-constraints-for-ca)
-   (key-usage
+  ([issuer-public-key
+    ca-public-key]
```

Comment:
I think we should give these schemas too

Reply:
Missed that, good catch. Updated.